sdk/ml/azure-ai-ml/samples/ml_samples_featurestore.py

```diff
@@ -0,0 +1,135 @@
+
+        featurestore_name = "my-featurestore"
+        featurestore_location = "eastus"
+        featurestore = FeatureStore(name=featurestore_name, location=featurestore_location)
+
+        # wait for featurestore creation
+        fs_poller = ml_client.feature_stores.begin_create(featurestore, update_dependent_resources=True)
+        print(fs_poller.result())
+        # [END create_feature_store]
+
+        featurestore_client = MLClient(
+            credential=credential,
+            subscription_id=subscription_id,
+            resource_group_name=resource_group,
+            workspace_name=featurestore_name,
+        )
+
+        # [START configure_feature_store_entity]
+        from azure.ai.ml.entities import DataColumn, DataColumnType
+        from azure.ai.ml.entities import FeatureStoreEntity
+
+        account_column = DataColumn(name="accountID", type=DataColumnType.STRING)
+
+        account_entity_config = FeatureStoreEntity(
+            name="account",
+            version="1",
+            index_columns=[account_column],
+            stage="Development",
+            description="This entity represents user account index key accountID.",
+            tags={"data_type": "nonPII"},
+        )
+
+        # wait for featurestore entity creation
+        fs_entity_poller = featurestore_client.feature_store_entities.begin_create_or_update(account_entity_config)
+        print(fs_entity_poller.result())
+        # [END configure_feature_store_entity]
+
+        # [START configure_feature_set]
+        from azure.ai.ml.entities import FeatureSetSpecification
+        from azure.ai.ml.entities import FeatureSet
+
+        transaction_fset_config = FeatureSet(
+            name="transactions",
+            version="1",
+            description="7-day and 3-day rolling aggregation of transactions featureset",
+            entities=["azureml:account:1"],
+            stage="Development",
+            specification=FeatureSetSpecification(
+                path="../tests/test_configs/feature_set/code_sample/FeatureSetSpec.yaml"
+            ),
+            tags={"data_type": "nonPII"},
+        )
+
+        feature_set_poller = featurestore_client.feature_sets.begin_create_or_update(transaction_fset_config)
+        print(feature_set_poller.result())
+        # [END configure_feature_set]
+
+        # [START configure_materialization_store]
+        from azure.ai.ml.entities import MaterializationStore
+
+        gen2_container_arm_id = "/subscriptions/{sub_id}/resourceGroups/{rg}/providers/Microsoft.Storage/storageAccounts/{account}/blobServices/default/containers/{container}".format(
+            sub_id=subscription_id,
+            rg=resource_group,
+            account=storage_account_name,
+            container=storage_file_system_name,
+        )
+
+        offline_store = MaterializationStore(
+            type="azure_data_lake_gen2",
+            target=gen2_container_arm_id,
+        )
```

Comment:
Seems offline_store is not used anywhere.

Reply:
I was trying to keep MaterializationStore sample succinct, but I can add usage of the offline_store if it'd be beneficial

Reply:
Yeah, @alanpo1, it doesn't necessarily need to be used in another call. If there's a function or other class that it's commonly used as an input to, then it could be nice to have one more line showing that, but otherwise it looks good as is.

Reply:
Added simple usage that shows common use case for MaterializationStore.